recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff 4*
         - libpng
         - openjpeg
         - curl
         - expat
         - zlib  # [not win]
-        - jpeg  # [not win]
+        - jpeg 9*  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5*  # [not win]
```

Comment:
To be honest I am completely lost. If you use `5.2*` it won't work on Python 3.5. If we use 5.0* it won't work on OS X.

In addition to that I start seem other failures yesterday that I plan to investigate today. (`gdal` was not finding `hdf4` and `hdf5` on Python 3.)